discuss/Formats.md

```diff
@@ -53,16 +53,23 @@ JSON is a fairly minimal format that lets you deliver sets of labeled data.  The
 ```JSON
 {
   "metadata": {
-    "lab":"Kerr",
-    "person":"Kerr",
+    "lab":{"location":"Janelia Research Campus", "name":"Kerr Lab" },
+    "who":"Kerr",
     "strain":"N2",
-    "date":"2015-04-27T15:15:32",
-    "units":"pixels"
+    "timestamp":"2015-04-27T15:15:32"
```

Comment:
I've changed it to a fictitious person if that works.  ([Janet Wallace](http://memory-alpha.wikia.com/wiki/Janet_Wallace))